backend/default.nix

```diff
@@ -1,9 +1,33 @@
-{ haskell-nix }:
+{ haskell-nix, ephemeralpg }:
 let
   project = haskell-nix.stackProject {
     src = haskell-nix.haskellLib.cleanGit {
       src = haskell-nix.cleanSourceHaskell { src = ./.; };
     };
+    modules = [{
+      doHaddock = true;
+      packages = {
+        edna = {
+          doHaddock = true;
+          ghcOptions = ["-Werror"];
+          components.tests.edna-test = {
+            ghcOptions = ["-Werror"];
```

Comment:
Do we need it both here and 2 lines above?

Reply:
As I can see in docs yes